example/lib/main.dart

```diff
@@ -5,7 +5,7 @@ import 'package:latlong/latlong.dart';
 import 'package:trufi_core/trufi_app.dart';
 import 'package:trufi_core/trufi_configuration.dart';
 
-void main() async {
+Future<void> main() async {
```

Comment:
why we need async function here?

Reply:
We wait directly for the globalCfg 
![grafik](https://user-images.githubusercontent.com/8026644/115229136-f8c3d900-a112-11eb-9934-0fe496782cec.png) (Line 12)

Reply:
ahh ok, I think we need to handle these async function possible errors, because if something goes wrong, the app will just show black screen.
for now I think is enough just add some FIXME comment

Reply:
Yeah, I guess in the long run I would love to get rid of the app_config file and more create a real class that can be overwritten, then we can add some code comments what which settings is etc. Also, it would be a bit more Typesafe.
I added a `TODO`